app/controllers/api_controller.rb

```diff
@@ -122,11 +122,11 @@ def set_access_control_headers
   end
 
   def base_config
-    Api::Settings.base
+    ManageIQ::API::Settings.base
```

Comment:
Squashed syntax has issues with constant lookup, see [Module.nesting section for a gotcha here](http://cirw.in/blog/constant-lookup.html)